client/src/components/Hours.jsx

```diff
@@ -4,39 +4,40 @@ import {Icon} from 'react-materialize';
 import moment from 'moment';
 
 
-const Hours = (props) => {
-  $(document).ready(function(){
-    $('.collapsible').collapsible();
-  });
-  console.log(props.hours)
+export default class Sidebar extends React.Component {
+  constructor(props) {
+    super(props);
 
- 
-  // find current day
-  let currDay = moment().weekday(); // Mon = 1, Tues = 2, etc.
-  let currHours = props.hours.slice(currDay-1).concat(props.hours.slice(0, currDay-1));
-  console.log(currHours)
+    this.state = {
+      currHours: [],
+      toggleHours: false
+    };
 
-  // check if now closed
+ 
+  };
+  sortHours() {
+    let currDay = moment().weekday(); // Mon = 1, Tues = 2, etc.
+    let currHours = props.hours.slice(currDay-1).concat(props.hours.slice(0, currDay-1));
```

Comment:
Needed some comments that could explain what you are planning to do with this part of code